api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/HivCohortDefinitionLibrary.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * The contents of this file are subject to the OpenMRS Public License
+ * Version 1.0 (the "License"); you may not use this file except in
+ * compliance with the License. You may obtain a copy of the License at
+ * http://license.openmrs.org
+ *
+ * Software distributed under the License is distributed on an "AS IS"
+ * basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See the
+ * License for the specific language governing rights and limitations
+ * under the License.
+ *
+ * Copyright (C) OpenMRS, LLC.  All Rights Reserved.
+ */
+package org.openmrs.module.eptsreports.reporting.library.cohorts;
+
+import java.util.Date;
+
+import org.openmrs.module.eptsreports.metadata.HivMetadata;
+import org.openmrs.module.reporting.cohort.definition.CohortDefinition;
+import org.openmrs.module.reporting.cohort.definition.SqlCohortDefinition;
+import org.openmrs.module.reporting.definition.library.BaseDefinitionLibrary;
+import org.openmrs.module.reporting.definition.library.DocumentedDefinition;
+import org.openmrs.module.reporting.evaluation.parameter.Parameter;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+/**
+ * Defines all of the Cohort Definition instances we want to expose for EPTS
+ */
+@Component
+public class HivCohortDefinitionLibrary extends BaseDefinitionLibrary<CohortDefinition> {
```

Comment:
Please move these sql cohort queries into the file grouping based on the type of query, as we discussed earlier today: Since these are SQL queries lets move these into `SqlCohortQueries.java`.

As we add more reports moving forward we can look into splitting these queries into their own files if the list of queries become too big.